feat(replays): add documentation on rage and dead clicks (#7591)
getsentry/sentry-docs: src/docs/product/session-replay/replay-page-and-filters.mdx

- **Duration**: The length of the replay. (Recordings can range from 1 second to 60 minutes.)
- **Errors:** The number of frontend errors identified by Sentry, as encountered in the replay. (Sometimes the error count reported is different from the actual errors captured. Read more in [Error Linking](/platforms/javascript/session-replay/#error-linking).)
- **Activity**: The activity level as determined by the number of errors encountered, their duration, and the number of UI events.

> - **Dead Clicks**: A user click that does not result in any page activity after 7 seconds (i.e. no HTML was added, removed, or updated; no visual changes were observed in the page.) They only affect `a` and `button` tags.

Reviewer: ```suggestion
- **Dead Clicks**: User clicks on `a` and `button` tags that do not result in any page activity after 7 seconds (i.e. no HTML was added, removed, or updated; no visual changes were observed in the page). 
```

- **Rage Clicks**: Five or more clicks on a dead element (it exhibits no page activity after 7 seconds.) Rage clicks are a subset of dead clicks.

> The minimum SDK version required to see rage and dead click data on the Replay index page is 7.60.1.

Reviewer: ```suggestion
SDK version `7.60.1` or higher is required to see rage and dead click data on the Replays page.
```
